scripts/lsp_types.lua

```diff
@@ -18,19 +19,90 @@ local function tofile(fname, text)
   end
 end
 
-function M.gen(opt)
-  if vim.uv.fs_stat('./lsp.json') then
-    vim.fn.delete('./lsp.json')
+local function read_json(opt)
+  local uri = 'https://raw.githubusercontent.com/microsoft/language-server-protocol/gh-pages/_specifications/lsp/'
+    .. opt.version
+    .. '/metaModel/metaModel.json'
+
+  local res = vim
+    .system({
+      'curl',
+      uri,
+      '-o',
+      '-',
+    })
+    :wait()
```

Comment:
is stylua forcing this? this is an excessive use of linebreaks.

```suggestion
  local res = vim.system({ 'curl', uri, '-o', '-'}):wait()
```

Reply:
yep. stylua force format . I need disable it on this script..